### PR TITLE
Make things easier to setup

### PR DESCRIPTION
**Pull request by crazyiop**
A few small things I changed who helped me to setup my keyboard. I guess it might help others.

### CODE REVIEW

**Review by kdb424 — CHANGES_REQUESTED** [review]

File: kmk/keys.py [hunk 1]
```diff
@@ -387,7 +387,7 @@ def __getitem__(self, key):
                 print(f'{key}: {maybe_key}')
 
             if not maybe_key:
-                raise ValueError('Invalid key')
+                raise ValueError(f'Invalid key: {key}')
```
[the remark this answers is not in the document]
Comment (crazyiop) [reply kmk/keys.py]: Moved that commit in a dedicated PR : https://github.com/KMKfw/kmk_firmware/pull/473

File: docs/porting_to_kmk.md [hunk 2]
```diff
@@ -63,6 +63,48 @@ coord_mapping = [
...
+Note: All number are not necessarily used ! The keyboard assume
+`number of line * number of rows` keys. Some of the possible keys might not be
+used. For example a keyboard with 60 keys might have 8 rows, 8 cols, allowing
+64 total combination hence 64 keys. 4 numbers will then not be used for keys
+in the `coord_mapping` (might be anyone of them depending of the wiring).
+
+### Find your coord mapping
+The following code will help you setup your `coord_mapping` by having every key
+send it's corresponding number. Use it after your pins and module definition
+to define both `keyboard.coord_mapping` and `keyboard.keymap`.
```
Comment (xs5871) [inline docs/porting_to_kmk.md:75]:
```suggestion
send its corresponding number. Use it after your pins and module definition
to define both `keyboard.coord_mapping` and `keyboard.keymap`.
```

File: docs/porting_to_kmk.md [hunk 3]
```diff
@@ -63,6 +63,48 @@ coord_mapping = [
...
+
+```python
+from kmk.handlers.sequences import simple_key_sequence
+from kmk.keys import KC
+
+# *2 for split keyboard, who will typically managed twice the number of keys
+# of one side. Having this N too much will have no impact (maybe slower boot..)
```
Comment (xs5871) [inline docs/porting_to_kmk.md:82]:
```suggestion
# *2 for split keyboards, which will typically manage twice the number of keys
# of one side. Having this N too large will have no impact (maybe slower boot...)
```

**Review by xs5871 — CHANGES_REQUESTED** [review]
There's a typo.

**Review by kdb424 — APPROVED** [review]

**Comment by kdb424** [comment]
I'll let @xs5871 merge when he's happy, but otherwise, I think it's good on my part.

**Review by xs5871 — APPROVED** [review]

**Pull request merged**